app/assets/stylesheets/application.css

```diff
@@ -88,6 +90,11 @@
   color: white;
 }
 
+.white-texxt {
+  font-size: 22px;
+  color: white;
+}
+
```

Comment:
I don't think the name of this css rule should have two x's in it. Makes it more challenging to remember what it is called and to reuse it.